_data/sites.json

```
@@ -3540,7 +3540,7 @@
         "name": "Coinbase",
         "url": "https://www.coinbase.com/settings/privacy-rights",
         "difficulty": "medium",
-        "notes": "Sign in then visit <a href=\"https://www.coinbase.com/settings/privacy-rights\">https://www.coinbase.com/settings/privacy-rights</a> and request the deletion of all of your data. You will then be directed to <a href=\"https://www.coinbase.com/settings/account_activity\">https://www.coinbase.com/settings/account_activity</a> where you must close your account after withdrawing all funds.",
+        "notes": "Sign in then visit [https://www.coinbase.com/settings/account_activity](https://www.coinbase.com/settings/privacy-rights\">https://www.coinbase.com/settings/privacy-rights</a> and request the deletion of all of your data. You will then be directed to <a href=\"https://www.coinbase.com/settings/account_activity) where you must close your account after withdrawing all funds.",
         "notes_tr": "Giriş yapın ve bağlantıyı ziyaret edin. Sayfanın altında hesabınızı kapatmak için bir seçenek göreceksiniz. Hesabı silmek için, hesabınızda bekleyen ödemelerin olmaması gerekir.",
```

```suggestion
        "notes": "Sign in then visit [their website](https://www.coinbase.com/settings/privacy-rights) and request the deletion of all of your data. You will then be directed to [their account activity page](https://www.coinbase.com/settings/account_activity) where you must close your account after withdrawing all funds.",
```
Let's delete the tr translation, since it is outdated.